src/Combinatorics/EnumerativeCombinatorics/combinations.jl

```diff
@@ -73,13 +73,17 @@ Base.length(C::Combinations) = binomial(C.n, C.k)
 
 Base.eltype(::Type{Combinations{T}}) where {T} = Combination{eltype(T)}
 
+function Base.show(io::IO, C::Combinations{Base.OneTo})
```

Comment:
Oh, I see you actually already do have a custom `show` method, excellent! It just doesn't get used... Presumably because you need this
```suggestion
function Base.show(io::IO, C::Combinations{<:Base.OneTo})
```